src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -328,6 +328,9 @@ class Microsoft::Terminal::Core::Terminal final :
     Microsoft::Console::Types::Viewport _mutableViewport;
     SHORT _scrollbackLines;
 
+    til::size _altBufferSize;

```

Comment:
Yeah that makes sense. In hindsight, it's obvious that it was easier to mess this up when both buffers were sharing.